beps/0002-dynamic-frontend-plugins/README.md

```diff
@@ -143,10 +143,16 @@ Plugin discovery is a pre-requisite for Plugin registry. This should be responsi
 
 ### Module federation implementation experiments
 
-> NOTE Share outcome of testing mixing multiple tools for module federation.
-
 Test should consist of trying to run permutations of webpack/Rspack/vite based shell apps/plugins and discover if we can freely choose any tool, or if we should restrict the tooling to just a subset of the available options.
 
+The outcome of initial testing is positive and it is possible to mix and match different build tools and consume different remote modules in a single shell application.
+
+The experimental code can be found in [this repository](https://github.com/scalprum/mf-mixing-experiments).
+
+**The testing so far was done only on very simple modules**. Although core React features are working (Context API and hooks), more testings needs to be done in order to declare this approach 100% reliable.
+
+So far a lot of custom code needs to be written to bridge Webpack, Rspack, @module-federation/enhanced with Vite. The first three are compatible out of the box, but Vite requires extra bridge to be able to consume/provide modules with/to other builds.
```

Comment:
👀, alright this bit is tricky, do you know if there are any initiatives on the Vite side to address this? Not blocking anything but important info to consider

Reply:
Not really. I can't find any mention of module federation support from the core development team. It is not in the milestone for v6 https://github.com/vitejs/vite/milestone/17, and not mentioned in the v5 milestone https://github.com/vitejs/vite/milestone/18 or the docs.

Every plugin is currently developed by third-party libraries. I am not saying we should not use third-party libraries, but the support for existing packages is questionable, and not all core module federation features are available. The missing support for registering dynamic modules at runtime([I've partially added here](https://github.com/scalprum/mf-mixing-experiments/blob/master/cross-utils/src/webpack-vite-bridge.ts)) is the main issue for our use case. There is the https://github.com/module-federation/vite which sounds "official" but there is not that much development happening here.